RichText-component/src/main/java/com/celements/rteConfig/RTEConfig.java

```diff
@@ -207,4 +217,15 @@ private static Optional<String> findFirstNonBlank(Stream<Supplier<Optional<Strin
         .findFirst();
   }
 
+  @Override
+  public @NotEmpty String jsRteUrl() {
+    return generateURL(":celRTE/tiny_mce_src.js");
+  }
+
+  private String generateURL(String rteUrlLink) {
+    return new AttachmentURLCommand().getAttachmentURL(
+        rteUrlLink, null, (String) null)
+        .map(UriComponents::toUriString).orElse(rteUrlLink);
```

Comment:
```suggestion
    return new AttachmentURLCommand()
        .getAttachmentURL(rteUrlLink, null, (String) null)
        .map(UriComponents::toUriString)
        .orElse(rteUrlLink);
```
A statement per line is more readable with fluent interfaces imo (if it doesn't all fit in one line).